### PR TITLE
Fix/add contributing md

### PR DESCRIPTION
**Pull request by bmw2621**
Fixed the reference to the project I stole this from and made it transferable to other projects by generalizing the linting rules.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
```
Comment (TheSavageDev) [inline CONTRIBUTING.md:13]: Nitpick: GitHub. Although this looks like copypasta

File: src/controllers/question.controller.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const httpStatus = require('http-status');
+const ApiError = require('../utils/ApiError');
```
Comment (TheSavageDev) [inline src/controllers/question.controller.js:2]: You aren't using either of these imports.

File: src/controllers/question.controller.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const { questionService } = require('../services');
+
+const getQuestions = async (res, res) => {
+  const result =  await questionService.queryQuestions();
+  return res.send(results)
+}
+
+module.exports = {
+  getQuestions
```
Comment (TheSavageDev) [inline src/controllers/question.controller.js:11]: Style decision: I prefer the comma dangle, but up to you guys.

File: src/models/question.model.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
```
Comment (TheSavageDev) [inline src/models/question.model.js:2]: Not using this import.

File: src/models/question.model.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+const questionSchema = mongoose.Schema(
+  {
+    author: {
+      type: String,
+      required: true,
```
Comment (TheSavageDev) [inline src/models/question.model.js:8]: Do we want to have this be a required key?

**Review by TheSavageDev — CHANGES_REQUESTED** [review]

File: src/controllers/question.controller.js (hunk 2 above)
Comment (bmw2621) [reply src/controllers/question.controller.js:2]: I didn't mean to stage anything but the contributing file before committing.  That was from me messing around.

**Pull request closed without merging**